pkg/openthread/contrib/platform_radio.c

```diff
@@ -138,8 +138,7 @@ void recv_pkt(otInstance *aInstance, netdev_t *dev)
     DEBUG("Openthread: Received pkt\n");
     netdev_ieee802154_rx_info_t rx_info;
     /* Read frame length from driver */
-    int len = dev->driver->recv(dev, NULL, 0, &rx_info);
-    Rssi = rx_info.rssi;
```

Comment:
You should fix the Samr21 transceiver driver if it sets that wrong value in the rx_info struct, not add a radio specific workaround in one of the consumers.